service/controller/clusterapi/v29/machine_deployment_resource_set.go

```diff
@@ -167,6 +170,18 @@ func NewMachineDeploymentResourceSet(config MachineDeploymentResourceSetConfig)
 		}
 	}
 
+	var tccpSubnetResource controller.Resource
+	{
+		c := tccpsubnet.Config{
+			Logger: config.Logger,
+		}
+
+		tccpSubnetResource, err = tccpsubnet.New(c)
```

Comment:
I had to add this resource to make the `clusterazs` resource get its shit together.